packages/cna-template/template/nuxt/components/Tutorial.vue

```diff
@@ -0,0 +1,46 @@
+<!-- Please remove this file from your project -->
+<template>
+  <div class="relative flex items-top justify-center min-h-screen bg-gray-100 sm:items-center sm:pt-0">
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
```

Comment:
```suggestion
    <link href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.1.2/tailwind.min.css" rel="stylesheet">
```

cdnjs is probably more stable for production usage.